src/groovy/org/pih/warehouse/core/RoleType.groovy

```diff
@@ -17,6 +17,7 @@ enum RoleType {
     ROLE_MANAGER('Manager', 2),
     ROLE_ASSISTANT('Assistant', 3),
     ROLE_BROWSER('Browser', 4),
+    ROLE_AUTHENTICATED('Authenticated', 5),
```

Comment:
I was expecting a ROLE_AUTHENTICATED as well as a ROLE_REQUESTOR and/or ROLE_CUSTOMER. 

ROLE_AUTHENTICATED is what a user gets by default if they log in and have no other primary roles. But a user should never be assigned ROLE_AUTHENTICATED. 

ROLE_REQUESTOR (and ROLE_CUSTOMER) would now be primary roles that inherit from ROLE_AUTHENTICATED. 

@pmuchowski Is that what you had in mind as well?

Reply:
I do not think I understand it well, I'll wait for a tech huddle to talk about it.

Reply:
@jmiranda, @MagdaToczek I think this is what we agreed on the tech huddle.
Add a new role -> Authenticated
Add a new complimentary role -> Requestor
Hence it is missing: `ROLE_REQUESTOR('Requestor, 100)` after `ROLE_INVOICE`